content/en/docs/instrumentation/js/exporters.md

```diff
@@ -140,7 +140,7 @@ accessible for everyone.
 
 It is recommended that you do not expose your collector directly, but that you
 put a reverse proxy (nginx, apache, ...) in front of it. The reverse proxy can
-take care of SSL-offloading, setting the right CORS headers and many other
+take care of SSL-offloading, and set the right CORS headers and many other
```

Comment:
I would revert this one, look at it like this:


The reverse proxy can take care of:
- SSL-offloading
- setting the right CORS headers
- and many other features...

Reply:
Yes, agreed